src/frontend/next/src/components/Posts/Timeline.tsx

```diff
@@ -0,0 +1,57 @@
+import { Container, createStyles, Grid } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import PostComponent from '../Post';
+import { Post } from '../../interfaces/index';
+import Spinner from '../Spinner';
+import LoadAutoScroll from './LoadAutoScroll';
+import useSiteMetaData from '../../hooks/use-site-metadata';
```

Comment:
Later we can switch this to use `src/config` directly instead of the hook.  This is fine for now.